bounties/README.md

```diff
@@ -5,6 +5,8 @@
 | :- | :- | :-
 | {x} XLM | {x} Credits | Capped ({x}) \| Continuos \| Competitive
 
+[🔵 Start hunting](https://github.com/tyvdh/stellar-quest-bounties/issues/new?assignees=&labels=&template=begin-the-hunt.md&title=%F0%9F%94%B5+%60{bounty-file-name}%60) this bounty.
+
 ## Context
 https://github.com/tyvdh/stellar-quest-bounties/issues?q={bounty-file-name}
```

Comment:
Let's actually bring both links here under the same umbrella. Maybe we remove the ## Context header and just float these two links. Let's put the second link under the title maybe `[View any existing entries](https://github.com/tyvdh/stellar-quest-bounties/issues?q={bounty-file-name})`

Great change!

Reply:
so like in #64?

Reply:
the second being the 'view existing' or 'start hunting'?